### ISSUE TITLE
Update to 3.7 fails with client authentication error on validator task

### ISSUE THREAD

**Issue by stzilli** [openshift/openshift-ansible#6941]
#### Description

Running the update script from 3.6 to 3.7 fails on `Confirm OpenShift authorization objects are in sync` task with client authentication error. This seems due to the fact the step is not using the `admin.kubeconfig` file compared to the other `oc adm` commands used in other parts of the code. 

##### Version

* Your ansible version per `ansible --version`

```
ansible 2.4.2.0
  config file = None
  configured module search path = [u'/xxx/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/Cellar/ansible/2.4.2.0_2/libexec/lib/python2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.10 (default, Oct 23 2015, 19:19:21) [GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)]
```

* The output of `git describe`

```
openshift-ansible-3.7.27-1-2-g823114205
```

##### Steps To Reproduce
1. Have a 3.6 Oringin installed
2. Launch the 3.7 upgrade script

##### Expected Results
The `Confirm OpenShift authorization objects are in sync` task should pass.

##### Observed Results
oc client authentication fails

```
TASK [Confirm OpenShift authorization objects are in sync] ***************************************************************************************************************************************************************************************************************************************************************************************
Tuesday 30 January 2018  14:40:20 +0100 (0:00:02.311)       0:03:38.045 ******* 
FAILED - RETRYING: Confirm OpenShift authorization objects are in sync (2 retries left).
FAILED - RETRYING: Confirm OpenShift authorization objects are in sync (1 retries left).
fatal: [mynode.xxx]: FAILED! => {"attempts": 2, "changed": false, "cmd": ["oc", "adm", "migrate", "authorization"], "delta": "0:00:00.243554", "end": "2018-01-30 13:40:53.832535", "msg": "non-zero return code", "rc": 1, "start": "2018-01-30 13:40:53.588981", "stderr": "", "stderr_lines": [], "stdout": "summary (dry run): total=0 errors=0 ignored=0 unchanged=0 migrated=0\nerror: exited without processing all resources: [the server has asked for the client to provide credentials (get clusterroles.authorization.openshift.io), the server has asked for the client to provide credentials (get roles.authorization.openshift.io), the server has asked for the client to provide credentials (get clusterrolebindings.authorization.openshift.io), the server has asked for the client to provide credentials (get rolebindings.authorization.openshift.io)]", "stdout_lines": ["summary (dry run): total=0 errors=0 ignored=0 unchanged=0 migrated=0", "error: exited without processing all resources: [the server has asked for the client to provide credentials (get clusterroles.authorization.openshift.io), the server has asked for the client to provide credentials (get roles.authorization.openshift.io), the server has asked for the client to provide credentials (get clusterrolebindings.authorization.openshift.io), the server has asked for the client to provide credentials (get rolebindings.authorization.openshift.io)]"]}
```

##### Additional Information

* Your operating system and version: CentOS Linux release 7.4.1708 (Core)

**Comment by joseorpa** [comment]
The same issue here, have you take any action to perform the upgrade?

**Comment by stzilli** [comment]
I've changed the failing command in `playbooks/common/openshift-cluster/upgrades/v3_7/validator.yml` adding `--config={{ openshift.common.config_base }}/master/admin.kubeconfig` (I think the admin.kubeconfig path was this one .. I hope my memory is not failing me) to have it authenticate as system:admin.

**Comment by sdodson** [comment]
/close
fixed via #8499
